### DISCUSSION TITLE
Different ways we can setup LLMStudio & their basic requirement

### DISCUSSION THREAD

**Discussion by sunilswain** [h2oai/h2o-llmstudio#711] [Q&A] [answered]
Hey I'd like to use h2o-llmstudio, I went to check for the setup instructions and seeing the basic requirement its too high. Is there any other ways I can set this up? Also I heard there is a cli version for this as well, what would be its basic requirement?

Thanks

**Comment by pascal-pfeiffer** [comment, marked as answer]
Hi @sunilswain ,
if you want to get started with H2O LLM Studio and don't have local hardware that meets the minimum requirements, make sure to check out the demo notebooks using free GPUs on google colab or kaggle:

> We offer several ways of getting started quickly.
> Using CLI for fine-tuning LLMs:

[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/code/ilu000/h2o-llm-studio-cli/) [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1soqfJjwDJwjjH-VzZYO_pUeLx5xY4N1K?usp=sharing)
